kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/consumer/ConsumerSeekUtil.java

```diff
@@ -50,11 +50,11 @@ private ConsumerSeekUtil() {
      *
      * @param consumer      a Kafka consumer instance
      * @param tokenSupplier a function that returns the current {@link KafkaTrackingToken}
-     * @param topics        a list of topics that will be assigned to the consumer
+     * @param subscriber    a {@link KafkaSubscriber} that contains the topics to subscribe to.
      */
     public static void seekToCurrentPositions(Consumer<?, ?> consumer, Supplier<KafkaTrackingToken> tokenSupplier,
-                                              List<String> topics) {
-        List<TopicPartition> all = topicPartitions(consumer, topics);
+                                              KafkaSubscriber subscriber) {
```

Comment:
This change is, sadly enough, a breaking change for users invoking this public method.
Hence, it would be better to add another method using the `KafkaSubscriber`, keeping the original signature intact.